pkg/mount/mount_test.go

```diff
@@ -79,6 +85,22 @@ var mountinfo = `39 21 0:36 / /tmp rw shared:91 -
 255 193 179:1 / /prefix/stage1/rootfs/opt/stage2/busybox/rootfs/rootfs/mnt/unknown ro,relatime -
 `
 
+// safeOrder checks for transitivity and (un)mount order sanity
```

Comment:
I'm not sure about this suggestion. While buggy, safeOrder just check the implementation for the sort interface. If we move this to use sort and equality comparison internally, it looks like we'll only check if the sort is stable and not if the result is actually correct. Am I missing something?

If so, I'd rather fix the off-by-one here. Not concerned about cubic complexity too much, as it is just a testing function and the set is ~50 entries.